index.js

```diff
@@ -18,16 +18,20 @@ if (typeof window === "undefined") {
     * @desc
     * Exports the [Repository CLI API client]{@link cmd} to access the CLI application
     * to interact from a shell with the Repository
+    *
+    * @example
+    * // Retrieve instance
+    * require('soasta-repository').cmd()
     */
-    exports.cmd = require("./cli/cmd").program;
+    exports.cmdRepository = require("./cli/cmd").program;
```

Comment:
So it looks like before we were exporting `cmd` and `CLI` multiple times, overwriting itself?  Thanks for fixing it.

@bobbykritzer @atomless will these export changes have any effect on the UI code?

Reply:
Assuming the team is using the API directly in their scripts, access to the CLI lib should not affect the teams work with the implementation. But will need confirmation just in case.

Reply:
@bobbykritzer @atomless would love your feedback above to confirm before we merge!

Reply:
@nicjansma, we no longer include this library in our dependencies. So the above change would not affect the new ui. That said - all looks good to me.